prediction/src/algorithms/segment/src/evaluate.py

```diff
@@ -19,7 +19,7 @@ def hausdorff_distance(ground_true, predicted):
     return hd
 
 
-def sensitivity(ground_true, predicted):
+def sensitivity(ground_true, predicted, smooth=1e-10):
```

Comment:
since this value is used a lot, it'd probably be better to create a top level constant

Reply:
Done. Good point, thanks!